R/authors-at-r.R

```diff
@@ -47,12 +47,54 @@ set_author_field <- function(authors, which, field, value) {
 
 
 ensure_authors_at_r <- function(obj) {
+  coerce_authors_at_r(obj)
   if (! obj$has_fields("Authors@R")) {
     stop("No 'Authors@R' field!\n",
          "You can create one with $add_author")
   }
 }
 
+coerce_authors_at_r <- function(obj) {
+  has_authors_at_r = obj$has_fields("Authors@R")
+  has_author = obj$has_fields("Author")
+  if (! (has_authors_at_r | has_author) ) {
+    stop("No 'Authors@R' or 'Author' field!\n",
+         "You can create one with $add_author")
+  }
+  if (has_author & has_authors_at_r) {
+    # Delete Author as it has Authors@R
+    obj$del("Author")
```

Comment:
I don't understand completely.  Can you not remove that field?  I can simply add the Authors@R field, but then the description will both have Author and Authors@R.  

I don't understand if you cannot modify the result within the function or if you do not want to use `del` in the function.  I removed any deletion in the recent pushes.  Is it OK if you do `self$set_authors` or is that modification undesirable as well?